lib/src/builder/generation/impl_generation.dart

```diff
@@ -400,8 +403,8 @@ class ImplGenerator {
             keyConstants[keyConstantName] = keyValue;
             constants[constantName] = constantValue;
 
-            final type = field.fields.type?.toSource();
```

Comment:
This was renamed so I could use the `type` argument above without it being shadowed here.

Reply:
you answered my feedback before i could leave it... damn you